config/deployment/dev/dispatch.yaml

```diff
@@ -0,0 +1,13 @@
+
+dispatch:
+  - url: "angio.dev.datadonationplatform.org/*"
+    service: angio
+  - url: "mbc.dev.datadonationplatform.org/*"
+    service: mbc
+  - url: "brain.dev.datadonationplatform.org/*"
+    service: brain
+  - url: "osteo.dev.datadonationplatform.org/*"
+    service: osteo
+  - url: "pepper-backend.dev.datadonationplatform.org/*"
+    service: pepper-backend
+
```

Comment:
It appears that GCP insists on the name of this file being dispatch.yaml . The command complains about the syntax of the file which led me to check again and again that I was not screwing up white spaces in the yaml. At any rate, there is one of these files for each environment.